jena-fuseki1/src/test/java/org/apache/jena/fuseki/ServerTest.java

```diff
@@ -20,7 +20,7 @@
 
 import java.io.IOException ;
 import java.net.ServerSocket ;
-import java.util.Arrays ;
+import java.util.concurrent.atomic.AtomicInteger;
```

Comment:
It can stay - I was just highlighting there is no concurrency issue at this point.